.travis.yml

```diff
@@ -11,24 +11,39 @@ env:
         - CI_MANAGERS="$TRAVIS_BUILD_DIR/travis-ci/managers"
         - VMTEST_DIR="$TRAVIS_BUILD_DIR/travis-ci/vmtest"
         - REPO_ROOT="$TRAVIS_BUILD_DIR"
-        # Default setup command run on VM boot.
-        - VMTEST_SETUPCMD='echo 42'
+        - VMTEST_SETUPCMD="PROJECT_NAME=${PROJECT_NAME} ./${PROJECT_NAME}/travis-ci/vmtest/run_selftests.sh"
     jobs:
         # Setup command override.
-        - KERNEL=5.4 VMTEST_SETUPCMD="PROJECT_NAME=${PROJECT_NAME} ./${PROJECT_NAME}/travis-ci/vmtest/setup_example.sh"
-        - KERNEL=5.3
-        - KERNEL=4.19.88
+        - KERNEL=5.5.0-rc6
+        - KERNEL=5.5.0
```

Comment:
`5.5.0-rc6` is built from `bpf-next` with pahole v1.16 and HEAD checked out to libbpf checkpoint commit.
So, technically, it's the latest kernel.
Debian still have pahole v.1.15 so more tests fail on kernel built in the CI. That's why there are both `LATEST` and `5.5.0-rc6`. Once v.1.16 is packaged in Deb, `5.5.0-rc6` can be removed.
`5.5.0` is built from main.
Sorry for confusion, I'll leave a comment.

Reply:
> So, technically, it's the latest kernel.
ah, right, it's because latest doesn't have pahole 1.16 yet... well, alright, but it seems like it might be better to just pull and build latest pahole as part of latest kernel compilation (maybe in a folllow up, don't block on that). Because we'll run into similar situations in the future, but we can't always wait for dwarves package to be updated in Ubuntu/Debian.